src/draw/drawRectangle.ts

```diff
@@ -56,14 +56,16 @@ export function drawRectangle(
   options: DrawRectangleOptions<Mask | Image> = {},
 ): Image | Mask {
   const {
+    width: rectangleWidth = image.width,
+    height: rectangleHeight = image.height,
     origin = { column: 0, row: 0 },
-    width = image.width,
-    height = image.height,
     strokeColor = getDefaultColor(image),
     fillColor = 'none',
   } = options;
-  const { column, row } = origin;
-
+  const width = Math.round(rectangleWidth ?? image.width);
+  const height = Math.round(rectangleHeight ?? image.width);
```

Comment:
```suggestion
  const width = Math.round(rectangleWidth);
  const height = Math.round(rectangleHeight);
```

You already set the default value above.

Reply:
You're right, I did it twice